docs/GHA.md

```diff
@@ -0,0 +1,50 @@
+
+The Buildkite Migration tool currently supports the following 
+
+- `runs_on`: The `runs_on` key defines the type of machine that the job will run on. Within Buildkite, this is mapped to an agent targeting [tag](https://buildkite.com/docs/agent/v3/queues#targeting-a-queue) of `runs_on`. Note that jobs that target custom `tag` names will have a `queue` target of `default`.
+- `steps`: Steps that are defined for a particular `job`. Any `run` key is supported - and at present, any action that defines a `uses` attribute currently is not supported.
+
+## Unsupported properties
+
+### defaults
+
+A `defaults` key allows the definition of a hash in which configuration will apply to all of a workflow's jobs. 
+
+While there isn't a direct transition from a GitHub Action workflow to a Buildkite pipeline - the Buildkite platform allows jobs to have default configuration applied to them through the use of [Agent](https://buildkite.com/docs/agent/v3/hooks#agent-lifecycle-hooks) or [Job](https://buildkite.com/docs/agent/v3/hooks#job-lifecycle-hooks) Lifecycle hooks. Agent lifecycle hooks allows the configuration of customized commands on agent start up and shut down. Job Lifecycle hooks allow for the customization of what occurs at each stage of a job's lifecycle - and can be specified and applied at an agent, repository or plugin level.
+
+### name
+
+The `name` key sets the name of the action as it will appear in the GitHub repositorys' "Actions" tab. When creating a Buildkite pipeline, it's name is set through the UI when first creating the pipeline - and can be altered within its pipeline settings, or via the [REST](https://buildkite.com/docs/apis/rest-api/pipelines#update-a-pipeline) or [GraphQL](https://buildkite.com/docs/apis/graphql/schemas/input-object/pipelineupdateinput) APIs.
+
+### on
+
+The `on` key allows for triggering a GitHub Action workflow. In Buildkite pipelines - this capability is defined within a `trigger` [step](https://buildkite.com/docs/pipelines/trigger-step) - where utilized within a pipeline, will create a build on the specified pipeline with additional properties.
+
+### permissions
+
+Within GitHub Action workflows, a `permissions` key allows the permission of the `GITHUB_TOKEN` to be modified with additional or removed access as required. Within Buildkite - [API Access Tokens](https://buildkite.com/docs/apis/managing-api-tokens) can be used within the context of a pipelines' build to interact with various Buildkite resources such as pipelines, artifacts, users, Test suites and more. Each token has a specified [token scope](https://buildkite.com/docs/apis/managing-api-tokens#token-scopes) that applies to interactions with the [REST](https://buildkite.com/docs/apis/rest-api) API, and can be configured with permission to interact with Buildkite's [GraphQL](https://buildkite.com/docs/apis/graphql-api) API.
```

Comment:
I would also clarify here that status check updates on Github are done through the application that must be connected and authorized for the integration to work and is not available for pipelines to use

Reply:
Added a snippet describing the GitHub App needs to be installed first for status checks (pertinent to the `permissions` allowing for the interaction of `statuses`